plugins/stockpiles/stockpiles.cpp

```diff
@@ -409,18 +409,7 @@ struct stockpiles_import_hook : public df::viewscreen_dwarfmodest
         auto dims = Gui::getDwarfmodeViewDims();
         int left_margin = dims.menu_x1 + 1;
         int x = left_margin;
-        int y = dims.y2 - 3;
-
-        int links = 0;
-        links += sp->links.give_to_pile.size();
-        links += sp->links.take_from_pile.size();
-        links += sp->links.give_to_workshop.size();
-        links += sp->links.take_from_workshop.size();
-        if ( links + 12 >= y )
-        {
-            y += 1;
-        }
-
+        int y = dims.y2 - 7; // above automelt, autotrade, stocks, and stockflow
```

Comment:
Unfortunately this overlaps with autodump now (it's not always enabled, but I guess I have it enabled?)

https://github.com/DFHack/dfhack/blob/eed1f2db6ae78440c6447e8abc4ca898ce3846d8/plugins/autodump.cpp#L215

It looks like there's one spot left above DF's `ESC: Done` row, so I'll put it there for now. Definitely something that we've considered resolving somewhere more centrally, though.

![image](https://user-images.githubusercontent.com/3719547/106372624-6e53b080-633f-11eb-9678-7f082d0753c6.png)

Reply:
What I ended up going with:
![image](https://user-images.githubusercontent.com/3719547/106372676-4749ae80-6340-11eb-85a2-db46537f2df1.png)